.github/workflows/sdk-cli-azure-test.yml

```diff
@@ -54,28 +54,31 @@ jobs:
         path: |
           ${{ github.workspace }}/src/promptflow/dist/*.whl
           ${{ github.workspace }}/src/promptflow-tools/dist/*.whl
-  sdk_cli_azure_test:
+
+  sdk_cli_azure_test_live:
+    if: github.event_name != 'pull_request'
     needs: build
     strategy:
       fail-fast: false
       matrix:
```

Comment:
MARK: offline synced with Philip, we may create another workflow for live test, as replay and live share little common lines.